File-Formats/PDF/Split-Documents.md

```diff
@@ -0,0 +1,276 @@
+
+Refer to the following code example to split a range of pages.
+
+{% tabs %}  
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+//Due to platform limitations, Essential PDF supports splitting a range of pages into separate PDF document only in Windows Forms, WPF, ASP.NET, and ASP.NET MVC platforms. However this can be achieved by using the following code snippet. 
+//Load the PDF document
+FileStream docStream = new FileStream("Input.pdf", FileMode.Open, FileAccess.Read);
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument(docStream);
+//Import the range of pages from the existing PDF
+loadedDocument.ImportPageRange(loadedDocument, 2, 8);
+
+//Create a memory stream 
+MemoryStream stream = new MemoryStream();
+//Save the document to stream
+loadedDocument.Save(stream);
+//Close the document
+loadedDocument.Close(true);
+
+{% endhighlight %}
+
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+int[,] values = new int[,] { { 2, 5 }, { 8, 10 } };
+//Load the PDF document
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument("Input.pdf");
+//Set a output path
+const string destinationFilePattern = "Output" + "{0}.pdf";
+//Split the pages into fixed number
+loadedDocument.SplitByRanges(destinationFilePattern, values);
+//close the document
+loadedDocument.Close(true);
+
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Load the PDF document.
+Dim loadedDocument As New PdfLoadedDocument("Input.pdf")
+'Set a output path
+Const destinationFilePattern As String = "Output" + "{0}.pdf"
+'Split the pages into fixed number.
+loadedDocument.SplitByRanges(destinationFilePattern, values)
+'Close the document.
+loadedDocument.Close(True)
+
+{% endhighlight %}
+
+{% endtabs %}  
+
+Download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PDF-Examples/tree/master/Split%20PDFs/Split-a-Range-of-Pages/.NET%20Framework).
+
+## Split by a fixed number of pages into a PDF document
+
+The Syncfusion .NET PDF library allows splitting by fixed number of pages of an existing PDF document using the [SplitByFixedNumber](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedDocument.html#Syncfusion_Pdf_Parsing_PdfLoadedDocument_SplitByFixedNumber_System_String_System_Int32_) method of the [PdfLoadedDocument](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Parsing.PdfLoadedDocument.html) class.
+Refer to the following code example to split by a fixed number of pages.
+
+{% tabs %}  
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//PDF doesn't support split by fixed number on the C#.NET Cross platforms.
```

Comment:
This method supports cross-platform too. You can use the following code to achieve that.

```
//Load the existing PDF file.
PdfLoadedDocument loadDocument = new PdfLoadedDocument(new FileStream("input.pdf", FileMode.Open));

//Subscribe to the document split event.
loadDocument.DocumentSplitEvent += LoadDocument_DocumentSplitEvent;

void LoadDocument_DocumentSplitEvent(object sender, PdfDocumentSplitEventArgs args)
{
    //Save the resulting document.
    FileStream outputStream = new FileStream(Guid.NewGuid().ToString() + ".pdf", FileMode.CreateNew);
    args.PdfDocumentData.CopyTo(outputStream);
    outputStream.Close();
}

//Spit the document by a fixed number.
loadDocument.SplitByFixedNumber(2);

//Close the document.
loadDocument.Close(true);
```